api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java

```diff
@@ -108,6 +115,7 @@
   private DbWorkspace workspace;
   private static final Instant NOW = Instant.now();
   private static final FakeClock CLOCK = new FakeClock(NOW, ZoneId.systemDefault());
+  private Map<Long, String> genderRaceEthnicityMap = new HashMap<>();
```

Comment:
you don't need to initialize it here if you're going to blow it away in setup.

Reply:
Opted for you suggestion of static `asMap` method on enum.